keras/losses.py

```diff
@@ -8,14 +8,19 @@
 from . import backend as K
 from .utils.generic_utils import deserialize_keras_object
 from .utils.generic_utils import serialize_keras_object
+import numpy as np
 
 
 def mean_squared_error(y_true, y_pred):
-    return K.mean(K.square(y_pred - y_true), axis=-1)
-
+    return np.square(np.subtract(y_pred,y_true)).mean() 
```

Comment:
2 things here:
- these functions are meant to return arrays of samplewise loss scores, not a single scalar. That's what we have the `axis=-1` argument.
- these functions are meant to operate on symbolic tensors, not Numpy arrays. That's why we use `K` methods instead of `np`.